src/main/scala/com/typesafe/webdriver/HtmlUnitWebDriverCommands.scala

```diff
@@ -16,13 +16,15 @@ class HtmlUnitWebDriverCommands() extends WebDriverCommands {
 
   import scala.concurrent.ExecutionContext.Implicits.global
 
-  override def createSession(): Future[String] = {
+
+  override def createSession(desiredCapabilities: JsObject = JsObject(),
+                             requiredCapabilities: JsObject = JsObject()): Future[(String, Either[WebDriverError, JsValue])] = {
```

Comment:
The return val seems like a complex structure. Would it benefit from being a case class?